src/core/list/components/Title.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
```

Comment:
you can import makeStyles from `@material-ui/core`

Reply:
Done.